docs/tutorial.rst

```diff
@@ -1,88 +0,0 @@
-DeepChem Tutorial
-=================
-
-If you're new to DeepChem, you probably want to know the basics. What is DeepChem? Why should you care about using it? The short answer is that DeepChem is a scientific machine learning library. (The "Chem" indicates the historical fact that DeepChem initially focused on chemical applications, but we aim to support all types of scientific applications more broadly).
-
-Why would you want to use DeepChem instead of another machine learning
-library? Simply put, DeepChem maintains an extensive collection of utilities
-to enable scientific deep learning including classes for loading scientific
-datasets, processing them, transforming them, splitting them up, and learning
-from them. Behind the scenes DeepChem uses a variety of other machine
-learning frameworks such as `sklearn`_, `tensorflow`_, and `xgboost`_. We are
-also experimenting with adding additional models implemented in `pytorch`_
-and `jax`_. Our focus is to facilitate scientific experimentation using
-whatever tools are available at hand.
-
-In the rest of this tutorials, we'll provide a rapid fire overview of DeepChem's API. DeepChem is a big library so we won't cover everything, but we should give you enough to get started.
-
-.. _`sklearn`: https://scikit-learn.org/stable/
-
-.. _`tensorflow`: https://www.tensorflow.org/
-
-.. _`xgboost`: https://xgboost.readthedocs.io/en/latest/
-
-.. _`pytorch`: https://pytorch.org/
-
-.. _`jax`: https://github.com/google/jax
-
-
-Quickstart
-----------
-If you're new, you can install DeepChem on a new machine with the following commands
-
-.. code-block:: bash
-
-    pip install tensorflow==2.3.0
-    pip install --pre deepchem
-
-
-DeepChem is under very active development at present, so we recommend using our nightly build until we release a next major release. Note that to use DeepChem for chemistry applications, you will have to also install RDKit using conda.
-
-.. code-block:: bash
-
-    conda install -y -c conda-forge rdkit
-
-
-
-Datasets
```

Comment:
I think it's still useful to have a basic tutorial in the docs itself. Colab is a little more heavy weight and it can be nice to give people a very quick glimpse of DeepChem code. Let's add this back into the new tutorial file. 

Ideally, we could expand this out so it's a self-contained micro-overview of all of DeepChem :)

Reply:
>  it can be nice to give people a very quick glimpse of DeepChem code

I think so, too! I will go back the dataset tutorials and add `featurizer`, `splitter` and `model` sections.